### ISSUE TITLE
[Bug]: Don't display the name of custom language i added by myself

### ISSUE THREAD

**Issue by abdelrhmanyosry** [pimcore/pimcore#16837]
### Pimcore version

11

### Steps to reproduce

adding custom languages in system_settings.yaml 
![image](https://github.com/pimcore/pimcore/assets/121692656/df1660e3-1a8d-439a-9932-8d3f38b26cd2)

then create any object with localize field you will find that the new lang i added have no name in UI.
![image](https://github.com/pimcore/pimcore/assets/121692656/efc20250-4249-4815-9d2a-e21c50dd31d0)

### Actual Behavior

it don't display the name of lang

### Expected Behavior

it should display the name of lang

**Comment by matthiashamacher** [comment]
Localized Fields are not intended to be used for anything else then localization with the available languages. That the name is not shown is because for Pimcore those are not valid languages and there are no translations and icons for that. If you want to show the name you would have to change more stuff in the core to make them visible. You can find the logic for the display here. There you can check what you would have to change to make it work. 

https://github.com/pimcore/admin-ui-classic-bundle/blob/b8de397f8b99e3ad0a36fd2e301faed9163ac2a9/public/js/pimcore/object/tags/localizedfields.js#L499-L517

Have you tried saving the object, did that work? You probably need to change even more things as the logic is tightly coupled with the languages. Probably best to use something else to achieve what you want.

**Comment by abdelrhmanyosry** [comment]
@matthiashamacher yeah it works and pimcore auto created tables for new localize fields.
![image](https://github.com/pimcore/pimcore/assets/121692656/a0df4226-557d-48aa-b98e-6f519253adb8)

what is your suggestion to able to send attributes to different store-views from magento to pimcore  vicevers?
